hamocc/aufr_bgc.F90

```diff
@@ -162,35 +162,35 @@ SUBROUTINE AUFR_BGC(kpie,kpje,kpke,ntr,ntrbgc,itrbgc,trc,               &
       ! Open netCDF data file
       !
       testio=0
-      IF(mnproc==1 .AND. IOTYPE==0) THEN
+      if (mnproc==1 .AND. IOTYPE==0) then
       
         ncstat = NF90_OPEN(rstfnm,NF90_NOWRITE, ncid)
-        IF ( ncstat .NE. NF90_NOERR ) THEN
-           CALL xchalt('(AUFR: Problem with netCDF1)')
+        if ( ncstat .NE. NF90_NOERR ) then
```

Comment:
maybe we could also replace .NE. with  /=, .EQ. with ==, and so forth?

Reply:
That would be very easy to do throughout the code.